typings/react-native-background-fetch/index.d.ts

```diff
@@ -0,0 +1,31 @@
+declare module 'react-native-background-fetch' {
```

Comment:
Wrote the typings here. Could contribute to DefinitelyTyped, need to look into that.